examples/audio/transformer_asr.py

```diff
@@ -0,0 +1,523 @@
+"""
+Title: Automatic Speech Recognition with Transformer
+Author: [Apoorv Nandan](https://twitter.com/NandanApoorv)
+Date created: 2021/01/13
+Last modified: 2021/01/13
+Description: Implement a complete transformer and train it on sequence to sequence ASR.
+"""
+"""
+## Introduction
+
+Automatic speech recognition (ASR) consists of transcribing audio segments into text.
```

Comment:
Suggestion:

```suggestion
Automatic speech recognition (ASR) tasks involve turning speech audio signals into text.
```

- Speech audio is not the same as music audio (the latter is MIR-related)